hazelcast/src/main/java/com/hazelcast/config/MapConfig.java

```diff
@@ -127,7 +127,9 @@
     private List<QueryCacheConfig> queryCacheConfigs;
     private PartitioningStrategyConfig partitioningStrategyConfig;
     private MetadataPolicy metadataPolicy = DEFAULT_METADATA_POLICY;
+    // todo: to be removed when we get rid of hot-restart
```

Comment:
Do we need this TODO? If we remove `HotRestartConfig`, the compiler will enforce us to remove this line.